priv/repo/migrations/20170317153551_create_message.exs

```diff
@@ -0,0 +1,15 @@
+defmodule Healthlocker.Repo.Migrations.CreateMessage do
+  use Ecto.Migration
+
+  def change do
+    create table(:messages) do
+      add :body, :string, null: false
+      add :name, :string
```

Comment:
name is the user's name (taken from the users table for that user) which I get from the `user_id` assigned to `socket`. The body is the content typed in by the user.